python_modules/libraries/dagster-looker/dagster_looker/asset_decorator.py

```diff
@@ -56,5 +75,6 @@ def looker_assets(*, project_dir: Path) -> Callable[[Callable[..., Any]], Assets
         specs=[
             *lookml_dashboard_specs,
```

Comment:
might be nice to pull out dashboard specs into their own little fn down the line, since I'd imagine they will get more complex down the line